aten/src/ATen/native/Distributions.cpp

```diff
@@ -126,71 +126,41 @@ DEFINE_DISPATCH(random_stub);
 DEFINE_DISPATCH(random_from_to_stub);
 DEFINE_DISPATCH(random_full_64_bits_range_stub);
 
+// ==================================================== Bernoulli =====================================================
+
+template<typename RNG>
+struct BernoulliStub {
+  void operator()(Tensor& self, const Tensor& p_, c10::optional<Generator> gen) {
+    bernoulli_tensor_stub(self.device().type(), self, p_, gen);
+  }
+
+  void operator()(Tensor& self, double p, c10::optional<Generator> gen) {
+    bernoulli_scalar_stub(self.device().type(), self, p, gen);
+  }
+};
+
 Tensor bernoulli(const Tensor& self, c10::optional<Generator> gen) {
-  return at::empty_like(self, LEGACY_CONTIGUOUS_MEMORY_FORMAT).bernoulli_(self, gen);
+  Tensor result = at::empty_like(self, MemoryFormat::Contiguous);
```

Comment:
Can you please replace it with, like it was before?
```suggestion
  Tensor result = at::empty_like(self, LEGACY_CONTIGUOUS_MEMORY_FORMAT);
```

Reply:
addressed